ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java

```diff
@@ -62,7 +62,7 @@ public SelenideElement getButton(String buttonTitle, SelenideElement differentRo
             .filter(Condition.exactText(buttonTitle)).shouldHave(sizeGreaterThanOrEqual(1));
 
         if (foundButtons.size() > 1) {
-            fail("Ambiguous button title. Found more that 1 button with title " + buttonTitle);
+            log.warn("Ambiguous button title. Found more that 1 button with title {}", buttonTitle);
```

Comment:
Nasty, but Apicurio has two save buttons when editing API Client, but both do the same.